src/lib.rs

```diff
@@ -1,165 +0,0 @@
-use std::{
-    process::Command,
-    thread,
-    fs,
-    path::{PathBuf, Path},
-    io::Write,
-    ffi::CString,
-    os::unix::net::UnixStream,
-    time,
-};
-
-use byteorder::{NativeEndian, ReadBytesExt, WriteBytesExt};
-use rand::{Rng, SeedableRng};
-
-macro_rules! cerr {
-    ($($arg:tt)*) => {{
-        Error::Err { msg: format!($($arg)*) }
-    }}
-}
-
-const PASSWORD : &str = "SUBPROCESS_INJECT_ENV__ARG__PASSWORD";
-const PASSWORD_LEN: usize = 32;
-
-const CONTROL_SOCK: &str = "SUBPROCESS_INJECT_ENV__ARG__CONTROL_SOCK";
-
-/// Errors encountered while setting up the environment injector.
-#[non_exhaustive]
-#[derive(Debug)]
-pub enum Error {
-    /// An opaque error with a useful debugging message but
-    /// which callers should not dispatch on.
-    Err {
-        msg: String,
-    },
-    __NonExhaustive,
```

Comment:
Why do we need this given the `non_exhaustive` attribute?

Reply:
Oh, maybe I can just leave it off.

Reply:
#5